packages/core/src/browser/quick-input/quick-command-frontend-contribution.ts

```diff
@@ -56,5 +61,10 @@ export class QuickCommandFrontendContribution implements CommandContribution, Ke
             command: quickCommand.id,
             keybinding: 'ctrlcmd+shift+p'
         });
+        keybindings.registerKeybinding({
+            command: 'workbench.action.closeQuickOpen',
+            keybinding: 'esc',
+            when: 'inQuickOpen'
+        });
```

Comment:
suggestion: we should add the 2nd keybinding available in vscode as well (<kbd>shift</kbd>+<kbd>esc</kbd>):

```suggestion
        });
        keybindings.registerKeybinding({
            command: 'workbench.action.closeQuickOpen',
            keybinding: 'shift+esc',
            when: 'inQuickOpen'
        });
```